kotest-assertions/kotest-assertions-livedata/src/androidMain/kotlin/io/kotest/assertions/livedata/getOrAwaitValue.kt

```diff
@@ -0,0 +1,39 @@
+package io.kotest.assertions.livedata
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+
+internal fun <T> LiveData<T>.getOrAwaitValue(
```

Comment:
Taken from Google examples; I was thinking of instead borrowing the TestObserver from https://github.com/jraska/livedata-testing/blob/master/testing/src/main/java/com/jraska/livedata/TestObserver.java and making it Kotlin since it's more complete and will allow for history-based assertions.